test/column-groups.html

```diff
@@ -530,7 +531,7 @@
           ].forEach(a => expect(getFooterCell(a[0], a[1]).hasAttribute('frozen')).to.be.true);
         });
 
-        it('should have last row fully visible when scrolled to end', done => {
+        (isFirefox ? it.skip : it)('should have last row fully visible when scrolled to end', done => {
```

Comment:
This is actually broken in Firefox: https://github.com/vaadin/web-components/issues/3717